domains/POAS/events/lpa-access-granted/index.md

```diff
@@ -0,0 +1,30 @@
+---
+name: lpa-access-granted
+version: 0.0.1
+summary: |
+  Actor granted access in Use An LPA
+producers:
+  - opg.poas.mrlpa
+consumers:
+  - opg.poas.use
+owners:
+  - use
+---
+
+## Context
+
+The LPA has been newly registered and the appropriate actors from MRLPA has been granted access in Use An LPA
+
+## Trigger
+
+Once user registration has been success (account created)
```

Comment:
Think this still need to be LPA registration, rather than user registration as MRLPA can only pass access on once an LPA is reaches registered status.